distributed/tests/test_joblib.py

```diff
@@ -11,17 +12,31 @@
 from distributed.utils_test import loop # noqa F401
 
 distributed_joblib = pytest.importorskip('distributed.joblib')
-joblibs = [distributed_joblib.joblib, distributed_joblib.sk_joblib]
 joblib_funcname = distributed_joblib.joblib_funcname
 
 
+@pytest.fixture(params=['joblib', 'sk_joblib'])
+def joblib(request):
+    if request.param == 'joblib':
+        try:
+            this_joblib = importlib.import_module('joblib')
```

Comment:
I went with `importlib` instead of an `import joblib` statement to avoid potential confusion about whether `joblib` referred to the imported module or the fixture.